internal/pkg/service/stream/storage/node/coordinator/slicerotation/operator.go

```diff
@@ -274,13 +275,18 @@ func (o *operator) rotateSlice(ctx context.Context, slice *sliceData) {
 	err = o.storage.Slice().Rotate(slice.SliceKey, o.clock.Now()).RequireLock(lock).Do(ctx).Err()
 	// Handle error
 	if err != nil {
-		o.logger.Errorf(ctx, "cannot rotate slice: %s", err)
-
-		// Increment retry delay
-		rErr := o.storage.Slice().IncrementRetryAttempt(slice.SliceKey, o.clock.Now(), err.Error()).RequireLock(lock).Do(ctx).Err()
-		if rErr != nil {
-			o.logger.Errorf(ctx, "cannot increment file rotation retry attempt: %s", err)
-			return
+		var stateErr sliceRepo.UnexpectedFileSliceStatesError
+		if errors.As(err, &stateErr) && stateErr.FileState != model.FileWriting {
+			o.logger.Info(ctx, "skipped slice rotation, file is already closed")
+		} else {
```

Comment:
Added check, slicerotation failed because it run in parallel with the filerotation, and file has been already rotated, all slices were closed.